connect/src/main/java/com/stripe/android/connect/EmbeddedComponentManager.kt

```diff
@@ -46,13 +47,19 @@ class EmbeddedComponentManager(
     appearance: Appearance = Appearance(),
     private val customFonts: List<CustomFontSource> = emptyList(),
 ) {
+
     private val _appearanceFlow = MutableStateFlow(appearance)
     internal val appearanceFlow: StateFlow<Appearance> get() = _appearanceFlow.asStateFlow()
 
-    private val logger = Logger.getInstance(enableLogging = BuildConfig.DEBUG)
+    private val isDebugBuild = BuildConfig.DEBUG
     private val loggerTag = javaClass.simpleName
 
-    private val isDebugBuild = BuildConfig.DEBUG
+    @Inject
+    internal lateinit var logger: Logger
+
+    init {
+        StripeConnectComponent.instance.inject(this)
```

Comment:
👀 Member injection isn't ideal, so this is just a stopgap to minimize the surface area of this PR. I have another coming up that will clean up `EmbeddedComponentManager` a lot.